src/Language/GraphQL/Draft/Syntax.hs

```diff
@@ -330,8 +335,7 @@ instance Hashable ValueConst
 
 data Value
   = VVariable !Variable
-  | VInt !Int32
-  | VFloat !Double
+  | VScientific !S.Scientific
```

Comment:
I think we should probably keep the `VInt` and `VFloat` cases, since they correspond to [Int values](http://spec.graphql.org/June2018/#sec-Int-Value) and [Float values](http://spec.graphql.org/June2018/#sec-Float-Value) in the GraphQL specification, which even have different syntax. Something like `123.0` should still be parsed as a Float value, not an Int value. The argument to `VInt` should probably become `Integer`, while the argument to `VFloat` should become `Scientific`.

Reply:
Okay, makes sense.